docs/user/overview.md

```diff
@@ -60,15 +60,15 @@ mv initial/install-config.yaml .
 rm -rf initial
 ```
 
-You can use the saved install-config for future clusters by copying it that install config into the asset directory and then invoking the installer:
+You can use the saved install-config for future clusters by copying it i nto the asset directory and then invoking the installer:
```

Comment:
"i nto" -> "into"

Reply:
argh, editing docs over a phone is dangerous.